scripts/configure.js

```diff
@@ -82,12 +82,19 @@ console.log(`Installing llnode for ${lldbExe}, lldb version ${lldbVersion}`);
 // TODO: The llvm project is probably moving to github soon at that point we
 // should stop using the mirror.
 if (lldbHeadersBranch != undefined) {
-  console.log('Cloning lldb from ' + lldbHeadersBranch);
-  child_process.execSync(`rm -rf ${lldbIncludeDir}`);
-  child_process.execFileSync('git',
+  if (!fs.lstatSync(path.join(lldbIncludeDir, 'include')).isDirectory()) {    
```

Comment:
`lstatSync()` throws if the path doesn't exist.

Reply:
Thanks for catching that (no pun intended)